### DISCUSSION TITLE
bluetooth

### DISCUSSION THREAD

**Discussion by dedi31** [blueman-project/blueman#2195] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

buon giorno non si connette nessuna periferica tramite bluetooth

**Comment by infirit** [comment]
https://github.com/blueman-project/blueman/wiki/Troubleshooting

Maybe ask on an Italian support channel for your distribution with general Bluetooth issues.
